github/ci/shared-deployments/files/docker-daemon-mirror-proxy.conf

```diff
@@ -0,0 +1,2 @@
+export http_proxy=http://docker-mirror-proxy.kubevirt-prow.svc:3128
+export https_proxy=http://docker-mirror-proxy.kubevirt-prow.svc:3128
```

Comment:
I think we also want to add:

```
DOCKER_OPTS="${DOCKER_OPTS} --data-root=/docker-graph"
DOCKER_OPTS="${DOCKER_OPTS} --mtu=1450"
DOCKER_OPTS="${DOCKER_OPTS} --ipv6"
DOCKER_OPTS="${DOCKER_OPTS} --fixed-cidr-v6 2001:db8:1::/64"
```

Reply:
We are currently merging `docker-daemon-mirror.conf` (which contains the options you mention) and `docker-daemon-mirror-proxy.conf` in the docker-proxy entry of the configmap, so that we don't need to repeat the settings in 2 different files https://github.com/kubevirt/project-infra/pull/717/files#diff-22cedcd0268e1984d6386801c0148fc967762c4e32bc56ab6d872a568122b11eR19 is that correct?

Reply:
ok, fine by me then. See it now.